test/lit.cfg

```
@@ -1340,6 +1340,76 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
+    config.target_add_rpath = ''
+    config.target_swift_frontend = ' '.join([
+        '%s', '-frontend', '-target %s',
+	'-Xcc --sysroot=%s', '-tools-directory %s',
+        '%s %s %s %s ',
+        ]) % (config.swift, config.variant_triple,
+           config.variant_sdk, tools_directory,
+           resource_dir_opt, mcp_opt,
+           config.swift_test_options, config.swift_frontend_test_options)
+    subst_target_swift_frontend_mock_sdk = config.target_swift_frontend
+    subst_target_swift_frontend_mock_sdk_after = ""
+    config.target_run = 'wasmer run --backend cranelift --'
+    if 'interpret' in lit_config.params:
+        use_interpreter_for_simple_runs()
+    config.target_sil_opt = (
+        '%s -target %s %s %s %s' %
+        (config.sil_opt, config.variant_triple, resource_dir_opt, mcp_opt, config.sil_test_options))
+    config.target_swift_symbolgraph_extract = ' '.join([
+        config.swift_symbolgraph_extract,
+        '-target', config.variant_triple,
+        mcp_opt])
+    config.target_swift_ide_test = (
+        '%s -target %s %s %s %s' %
+        (config.swift_ide_test, config.variant_triple, resource_dir_opt,
+         mcp_opt, ccp_opt))
+    subst_target_swift_ide_test_mock_sdk = config.target_swift_ide_test
+    subst_target_swift_ide_test_mock_sdk_after = ""
+    config.target_swiftc_driver = (
+        "%s -target %s -toolchain-stdlib-rpath %s %s" %
+        (config.swiftc, config.variant_triple, resource_dir_opt, mcp_opt))
+    config.target_swift_modulewrap = (
+        '%s -modulewrap -target %s' %
+        (config.swiftc, config.variant_triple))
+    config.target_swift_emit_pcm = (
+        '%s -emit-pcm -target %s' %
+        (config.swiftc, config.variant_triple))
+    config.target_clang = (
+        "clang++ -target %s %s -fobjc-runtime=ios-5.0" %
+        (config.variant_triple, clang_mcp_opt))
+    config.target_ld = "ld -L%r" % (make_path(test_resource_dir, config.target_sdk_name))
```

Please do not use `ld`.  This is going to fallback to random linkers.  Please use the absolute path to `ld.wasm` that you have found and logged.